index.ts

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+const todo = require("./src/todo");
```

Comment:
Normally it is good practice to cluster all your requires at the top of the page.  This allows you to only require once and to easily eliminate duplicates (if you sort the order).